packages/framework/src/Model/Order/Withdrawal/WithdrawalFacade.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Order\Withdrawal;
+
+use DateTimeInterface;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Setting\Setting;
+use Shopsys\FrameworkBundle\Model\Order\Order;
+use Shopsys\FrameworkBundle\Model\Order\OrderUrlGenerator;
+use Shopsys\FrameworkBundle\Model\Order\Withdrawal\Exception\WithdrawalException;
+
+class WithdrawalFacade
+{
+    public const string VARIABLE_ORDER_DETAIL_URL = '{order_detail_url}';
+    public const string VARIABLE_ORDER_NUMBER = '{order_number}';
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     * @param \Shopsys\FrameworkBundle\Component\Setting\Setting $setting
+     * @param \Shopsys\FrameworkBundle\Model\Order\OrderUrlGenerator $orderUrlGenerator
+     * @param \Shopsys\FrameworkBundle\Model\Order\Withdrawal\WithdrawalChecker $withdrawalChecker
+     * @param \Shopsys\FrameworkBundle\Model\Order\Withdrawal\WithdrawalDeadlineCalculation $withdrawalDeadlineCalculation
+     */
+    public function __construct(
+        protected readonly Domain $domain,
+        protected readonly Setting $setting,
+        protected readonly OrderUrlGenerator $orderUrlGenerator,
+        protected readonly WithdrawalChecker $withdrawalChecker,
+        protected readonly WithdrawalDeadlineCalculation $withdrawalDeadlineCalculation,
+    ) {
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Order\Order $order
+     * @return string
+     */
+    public function getWithdrawalInstructions(Order $order): string
+    {
+        $withdrawalInstructions = $this->setting->getForDomain(
+            Setting::WITHDRAWAL_INSTRUCTIONS,
+            $this->domain->getId(),
```

Comment:
🤔 do we want the instructions for the current domain, or the domain of the order?